book/src/super-sql/declarations/operators.md

```diff
@@ -43,29 +43,29 @@ In contrast to function calls, where the arguments are evaluated at the call sit
 and values are passed to the function, operator arguments are instead passed to the
 operator body as an expression _template_ and the expression is evaluated in the
 context of the operator body.  That said, any other declared identifiers referenced
-by these expressions (e.g., constants, functions, named queries, etc.) are bound to
+by these expressions (e.g., [constants](constants.md), [functions](functions.md), [named queries](queries.md), etc.) are bound to
 those entities using the lexical scope where they are defined rather than the
 scope of their expanded use in the operator's definition.
 
 These expression arguments can be viewed as a
 [closure](https://en.wikipedia.org/wiki/Closure_(computer_programming))
 though there is no persistent state stored in the closure.
 The [jq](https://github.com/jqlang/jq/wiki/jq-Language-Description#the-jq-language) language
-describes its expression semantics as closures as well, though unlike jq,
+describes its expression semantics as closures as well, though unlike `jq`,
```

Comment:
I feel like this reference is to jq the system, not `jq` the command.  Kind of like `super` vs SuperDB.  Wouldn't say `SuperDB`.  Or `clickhouse` the command vs ClickHouse the system.